.github/workflows/ci.yml

```diff
@@ -0,0 +1,51 @@
+name: CI
+on:
+  pull_request:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+jobs:
+  CI:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      # Checkout the branch
+      - uses: actions/checkout@v3
+      # Java is needed for the Scala Play app
+      - uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'corretto'
+          cache: 'sbt'
+      # Build CDK and Play (in sequence)
+      - run: sbt clean compile test assembly
```

Comment:
I was looking at the other [geoip-db-refresher](https://github.com/guardian/ophan-geoip-db-refresher/pull/256) PR and it looks different in this step which you changed in the `ci.sh` file but it doesn't have the word `test`. Why is it different?

Reply:
Good question, simple answer, [geoip-db-refresher](https://github.com/guardian/ophan-geoip-db-refresher) has no tests!